apps/console/src/features/organizations/components/organization-switch/organization-switch-breadcrumb.tsx

```diff
@@ -75,6 +75,13 @@ export const OrganizationSwitchBreadcrumb: FunctionComponent<OrganizationSwitchD
         shouldSendRequest
     );
 
+    const isSubOrg: boolean = window[ "AppUtils" ].getConfig().organizationName;
+
+    const isShowSwitcher: boolean = (
+        (organizationConfigs?.showSwitcherInTenants && !isSubOrg) ||
```

Comment:
Shall we improve readability here?

 const isShowSwitcher: boolean = (
        (
            organizationConfigs?.showSwitcherInTenants
            && !isSubOrg
        ) || (
            !organizationConfigs?.showSwitcherInTenants
            && isSubOrg
        )